src/api/audits.coffee

```diff
@@ -0,0 +1,129 @@
+Audit = require('../model/audits').Audit
+authorisation = require './authorisation'
+Q = require 'q'
+logger = require 'winston'
+utils = require "../utils"
+
+
+# function to construct projection object
+getProjectionObject = (filterRepresentation) ->
+  switch filterRepresentation
+    when "simpledetails"
+      # view minimum required data for audit details view
+      return {}
+    when "full"
+      # view all audit data
+      return {}
+    else
+      # no filterRepresentation supplied - simple view
+      # view minimum required data for audits
+      return { "participantObjectIdentification": 0, "activeParticipant": 0, "rawMessage": 0 }
```

Comment:
A good field to exclude would be [participantObjectQuery](https://github.com/jembi/openhim-core-js/blob/master/src/model/audits.coffee#L24). That contains the actual message (pix, xds) that was sent. In fact we should probably project that out for all cases except `full`.